Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -2501,6 +2534,22 @@ void webkitWebViewRunAsModal(WebKitWebView* webView)
 #endif
 }
 
+bool webkitWebViewHasMainResource(WebKitWebView* webView)
+{
+    ASSERT(webView != nullptr);
```

Comment:
Omit this (it's not used anywhere else in this file, and callers know not to pass nullptr to method calls).